javalib/src/main/scala-2/java/lang/Enum.scala

```diff
@@ -1,3 +1,6 @@
+// Classes in this file needs special handling in Scala 3, we need to make sure
```

Comment:
```suggestion
// Classes in this file need special handling in Scala 3, we need to make sure
```
Typo, here and in other scala-2 files